src/Components/SortControl/SortControl.module.css

```diff
@@ -0,0 +1,29 @@
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+    'Droid Sans', 'Helvetica Neue', sans-serif;
+}
+
+.container > span {
+  color: #979797;
+  margin-right: 10px;
+}
+
+.container > select {
+  font-size: 16px;
+  color: white;
+  background-color: transparent;
+  border: none;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+    'Droid Sans', 'Helvetica Neue', sans-serif;
+}
+
+option {
+  background-color: #333;
+}
+
+option:hover {
+  background-color: #d94b52;
+}
```

Comment:
Again, set class names to everything. Do not increase specificity with something like ` > `, and even use class names instead of tag like `option`.